Onion.Infrastructure/Services/LoggerService.cs

```diff
@@ -0,0 +1,33 @@
+﻿using Onion.Core.Interfaces.Services;
+using Onion.Core.Models;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Xml.Linq;
+
+namespace Onion.Infrastructure.Services
+{
+    public  class LoggerService : ILoggerService
+    {
+        public Error GetError(string id)
+        {
+            //2020-10-06
+
+            var xml = "<log>" + System.IO.File.ReadAllText(@$"C:\Users\Victor Ayala\source\repos\OnionArchitecture\OnionArchitecture\Onion\bin\Debug\netcoreapp3.1\Logs\{DateTime.Now.ToString("yyyy-MM-dd")}.xml") + "</log>";
```

Comment:
It's pretty rare that we should hard code a path. At best it should be in some sort of configuration file. We let some of this slide in tests (look up the logging test I updated). But this wouldn't work on anyone's machine but Victors...

Reply:
Working on it